js/foundation.offcanvas.js

```diff
@@ -205,6 +206,12 @@ class OffCanvas {
       window.scrollTo(0,document.body.scrollHeight);
     }
 
+    if (this.options.transitionTime && this.options.transition !== 'overlap') {
+      this.$element.siblings('[data-off-canvas-content]').css('transition-duration', this.options.transitionTime);
+    } else {
+      this.$element.siblings('[data-off-canvas-content]').css('transition-duration', '');
```

Comment:
Yep the variable still exists. However if a user adds the data attribute, that value overrules it. Try out the visual test if you can, that will demonstrate it better than I can explain :)